modules/kde/hm.nix

```diff
@@ -225,7 +225,16 @@ in {
     config.lib.stylix.mkEnableTarget "KDE" pkgs.stdenv.hostPlatform.isLinux;
 
   config = lib.mkIf config.stylix.targets.kde.enable {
-    home.packages = [ themePackage ];
+    home.packages = [ themePackage pkgs.breeze-icons ];
+
+    qt = {
+      enable = true;
+      style.name = "breeze";
+      platformTheme = "kde";
+    };
+
+    xdg.configFile."kdeglobals".text = "${formatConfig colorscheme}";
```

Comment:
Unfortunately, we cannot simply overwrite `kdeglobals`, since this needs to be writeable for some other KDE settings. Instead, settings should be added to the `kdeglobals` attribute set earlier in the file, using `makeImmutable` to prevent each individual setting being changed imperatively.

Reply:
KDE should still show the existence of the stylix theme in the settings menu - I'm not sure I fully agree that this is a measurably worse experience since I don't see how a KDE user using stylix would be looking at the system theme settings anyways when these are configured by stylix and can't reliably be changed through KDE.

I tried doing it through the other `kdeglobals` file but it seemed to have no effect. Doing it this way would also cause some other issues - dolphin, for example, seemingly reads directly from `.config/kdeglobals` when setting the background color for  the window, rather than doing it through qt.

What other `kdeglobals` settings are set in there anyways? The `plasma-apply-lookandfeel` package directly overwrites `kdeglobals` anyways, so I struggle to see how there are other configuration options in KDE that are there that wouldn't also get overwritten when using `plasma-apply-lookandfeel`.

Reply:
From a GitHub search, it appears to be used for various other settings:

https://github.com/rockerBOO/dotfiles/blob/ad8c4c43d8e9279f3a97bf9b4995639e026bf7b4/config/kdeglobals#L4-L5
https://github.com/ZanSara/dotfiles/blob/4af485809284e621ea60aa5b0970d272a096fd1d/.config/kdeglobals
https://github.com/lliurex/lliurex-artwork/blob/e34b5ca2348a05bcdffe0f45a3be59e033c65316/defaults/xdg/lliurex/desktop/kdeglobals#L4-L5

Reply:
How should we go about doing this then? I think I need to try and understand more about what `kdeglobals` actually does - either that or figure out how to get the other `kdeglobals` file. Or should we just do it through `plasma-workspace`?